frappe/__init__.py

```diff
@@ -1920,16 +1920,20 @@ def logger(
 	)
 
 
-def get_desk_link(doctype, name, show_title_with_name=False):
+def get_desk_link(doctype, name, show_title_with_name=False, open_in_new_tab=False):
 	meta = get_meta(doctype)
 	title = get_value(doctype, name, meta.get_title_field())
 
+	target_attr = ' target="_blank"' if open_in_new_tab else ""
+
 	if show_title_with_name and name != title:
-		html = '<a href="/app/Form/{doctype}/{name}" style="font-weight: bold;">{doctype_local} {name}: {title_local}</a>'
+		html = '<a href="/app/Form/{doctype}/{name}"{target} style="font-weight: bold;">{doctype_local} {name}: {title_local}</a>'
 	else:
-		html = '<a href="/app/Form/{doctype}/{name}" style="font-weight: bold;">{doctype_local} {title_local}</a>'
+		html = '<a href="/app/Form/{doctype}/{name}"{target} style="font-weight: bold;">{doctype_local} {title_local}</a>'
```

Comment:
Looks good to me

---
[not relevant to your PR]

Just noticed the pretty obvious HTML injection here :smile:. There is potential for bad formatting (and malicious usage too).

```
title_local is an attacker-controlled variable via the document title
doctype_local is System Manager-controlled variable via Custom Translation
```